internal/pkg/service/buffer/store/file.go

```diff
@@ -71,31 +70,6 @@ func (s *Store) getFileOp(_ context.Context, fileKey key.FileKey) op.ForType[*op
 		})
 }
 
-func (s *Store) ListFilesInState(ctx context.Context, state filestate.State, exportKey key.ExportKey, ops ...iterator.Option) (files []model.File, err error) {
-	_, span := s.tracer.Start(ctx, "keboola.go.buffer.store.ListFilesInState")
```

Comment:
Tieto jednoucelove metody su uz priamo v kode, ono to nema zmysel velmi kopirovat vsetko do pamate, ked sa mozu dane operacie vykonat priamo vo `ForEachValue`.

Reply:
Vid `cleanup.Task`, je tam iba priame volanie `ForEach`, nic naviac co by malo zmysel dat do samostatnej metody a testovat.